static/src/stylesheets/module/content/_media-player.scss

```diff
@@ -20,9 +20,13 @@ $ima-controls-height: 70px;
     -webkit-transform: translateZ(0); // fixes iOS hover bug
 }
 .gu-media-wrapper--audio {
-    background-color: $vjs-control-colour;
+    background-color: rgba($vjs-control-colour, .8);
```

Comment:
And for a few [further](https://github.com/guardian/frontend/pull/12846/files#r62980282) [down](https://github.com/guardian/frontend/pull/12846/files#diff-bb3768ac4b8bd3b37670fc613c32ffebR201)